bindata/network/ovn-kubernetes/managed/ovnkube-control-plane.yaml

```diff
@@ -174,6 +174,14 @@ spec:
             persistent_ips_enabled_flag="--enable-persistent-ips"
           fi
 
+          # This is needed so that converting clusters from GA to TP
+          # will rollout control plane pods as well
+          network_segmentation_enabled_flag=
+          if [[ "{{.OVN_NETWORK_SEGMENTATION_ENABLE}}" == "true" ]]; then
+            multi_network_enabled_flag="--enable-multi-network"
```

Comment:
`multi_network_enabled_flag` is not set when `OVN_NETWORK_SEGMENTATION_ENABLE` is not true

Reply:
ofc it isn't (Facepalm), some features got the free pass to be GA without feature gates huh.
nice catch pat!